pkg/queue/local_queue.go

```diff
@@ -35,23 +35,25 @@ func DefaultQueueKey(namespace string) string {
 
 // LocalQueue is the internal implementation of kueue.LocalQueue.
 type LocalQueue struct {
-	Key          string
-	ClusterQueue string
-
-	items map[string]*workload.Info
+	Key           string
+	ClusterQueue  string
+	LocalQueueObj *kueue.LocalQueue
+	items         map[string]*workload.Info
 }
 
 func newLocalQueue(q *kueue.LocalQueue) *LocalQueue {
 	qImpl := &LocalQueue{
-		Key:   Key(q),
-		items: make(map[string]*workload.Info),
+		Key:           Key(q),
+		LocalQueueObj: q,
```

Comment:
```suggestion
		labels: q.ObjectMeta.Labels,
```

I do not think we want to store localQueue object.